README.md

```diff
@@ -393,6 +393,22 @@ Debug.prettyPrint(label: "array", array)
 Note:
 This can't be used to the operator-based API such as `p >>>`. (This is a Swift language's limitation)
 
+### Debug Settings
+It is recommended to move `.lldbinit` to `~/.lldbinit` to easier to debugging with SwiftPrettyPrint.
```

Comment:
I tried rewrote it so that I could understand it immediately after reading it, how about it?

```suggestion
Please copy and add follows to your `~/..ldbinit` (please create file when nothing):

``text
command regex _p 's/(.+)/e - l swift -o -- Pretty.print(%1)/'
command regex _pp 's/(.+)/e -l swift -o -- Pretty.prettyPrint(%1)/'
``

```

(Please change `` to ```)

Reply:
fixed it!